ada/ast.py

```diff
@@ -1502,17 +1519,19 @@ def get_aspect(name=Symbol):
 
         This will return the syntactic node corresponding to attribute
         directly.
+
+        Note: for some aspects (e.g. Inline), Libadalang will check if they are
+        defined on any part of the entity.
         """
-        return (
-            Entity.get_pragma(name).then(
-                lambda p: Aspect.new(
-                    exists=True, node=p, value=p.args._.at(1)._.assoc_expr
-                )
-            )._or(Entity.get_aspect_assoc(name).then(
-                lambda aa: Aspect.new(exists=True, node=aa, value=aa.expr)
-            ))._or(Entity.get_representation_clause(name).then(
-                lambda rc: Aspect.new(exists=True, node=rc, value=rc.expr)
-            ))
+        parts_to_check = Var(If(
+            name.any_of('Inline'),
```

Comment:
At least put a comment here saying that this list is meant to be grown when we add more pragmas/aspects that behave like this ... As is it's clearly very weird :)